fedora_messaging/exceptions.py

```diff
@@ -17,7 +17,7 @@ class PermissionException(BaseException):
         obj_type (str): The type of object being accessed that caused the
             permission error. May be None if the cause is unknown.
         description (object): The description of the object, if any. May be None.
-        reason (str): The reason the server gave for the permission error, if
+        reason (str): The reason the server gave the permission error, if
```

Comment:
The `for` seems justified for me, it's the reason for the error, no?

Reply:
I misread it. Sometimes I don't understand English. I'm reverting it.